library/src/morpheus-master/sass/modules/tool/sitestats/_sitestats.scss

```diff
@@ -215,469 +221,399 @@
-    color: #777;
+    color: var(--sakai-text-color-2);
   }
 
-
   /** Tool icon */
   .toolIcon {
     padding-left: 23px;
     background-repeat: no-repeat;
     background-position: left center;
   }
 
-
   /** Summary Header separator */
   .summaryHeader {
-    border-bottom: 2px solid #DDDFE1;
-    color: #333;
+    border-bottom: 2px solid var(--sakai-border-color);
+    color: var(--sakai-text-color-1);
     margin: 5px 0 0 0;
     padding: 5px 0 1px 2px;
     font-size: 15px;
     font-weight: bold;
   }
+
   .summaryInstruction {
     padding: 4px;
-    color: #555;
+    color: var(--instruction-color);
     font-size: 11px;
     font-weight: normal;
   }
+
   .summaryHeader span.itemAction {
     font-size: 11px;
   }
 
-
   /** Form */
   .formContainer {
     padding-left: 2px;
     margin: 0;
   }
+
   table.formContainer  {
     margin: 12px 0 12px 0;
     padding-left: 0px;
   }
+
   .formContainer td {
     display: table-cell;
     line-height: 24px;
     margin: 0;
     padding: 2px 0px 2px 0px;
   }
+
   .formContainer td.lbl {
     padding: 0px 16px 0px 0px;
     font-weight: bold;
     text-align: right;
     width: 150px;
     vertical-align:middle;
   }
+
   .formContainer select{
     margin: 0.1em 0.3em;
-    border:1px solid #888888;
+    border: 1px solid var(--sakai-border-color);
   }
+
   .formContainer input[type="text"], .formContainer textarea {
     padding:0.1em 0.3em;
     margin: 0.1em 0.3em;
-    border:1px solid #888888;
+    border: 1px solid var(--sakai-border-color);
   }
+
   .formContainer .responsive-form {
     width: 300px;
   }
+
   @media screen  and (max-width: 580px) {
     .formContainer .responsive-form {
       width: 100%;
     }
   }
 
-
   /** Summary Table */
   .summaryTable {
-    /*border: 1px dashed #cccccc;*/
     text-align: left;
-    /*margin-top: 20px;*/
     width: 100%;
   }
+
   .summaryTable .labelColumn {
     min-width: 150px;
-    /* IE Dynamic Expression to set the width, ignored by other browsers */
-    width: expression(document.body.clientWidth < 300 ? "150px" : "auto" );
   }
+
   label.indnt0 {
     font-weight: bold;
   }
+
   label.indnt1 {
     font-weight: normal;
   }
+
   .selector {
-    color: #555555;
+    color: var(--sakai-text-color-2);
     font-size: 85%;
     font-weight: normal;
     margin: 0pt;
   }
+
   .additionalInfo {
-    color: #555555;
+    color: var(--sakai-text-color-2);
     font-size: 85%;
     font-weight: normal;
     margin: 0pt;
     margin-top: 3px;
     float: right;
   }
+
   .description {
-    color: #333333;
+    color: var(--sakai-text-color-1);
     font-size: 100%;
     font-weight: normal;
     margin: 0pt;
   }
+
   .notes {
-    color: #333333;
+    color: var(--sakai-text-color-1);
     font-size: 90%;
     font-weight: normal;
     margin: 0pt;
   }
 
-
-  /** Ajax loading */
-  .ajaxLoading {
-    background: #ffffff url(/sitestats-tool/images/ajaxload.gif) no-repeat;
-    color: window;
-  }
-
-
```

Comment:
Is this not needed?

Reply:
I didn't think so but Shawn mentioned it might be needed by wicket elsewhere, so i'm putting it back in.